content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,233 @@
+
+- `YOUR-PERSONAL-API-TOKEN`
+- `YOUR-V2-ORGANIZATION-TOKEN` 
+
+You can create an [asset](../../getting-started/glossary/#asset) with the following REST call:
+
+```bash
+curl -X POST "https://api.cobalt.io/assets" \
+  -H 'Accept: application/vnd.cobalt.v2+json' \
+  -H 'Authorization: Bearer YOUR-PERSONAL-API-TOKEN' \
+  -H 'Content-Type: application/vnd.cobalt.v2+json' \
+  -H 'Idempotency-Key: A-UNIQUE-IDENTIFIER-TO-PREVENT-UNINTENTIONAL-DUPLICATION' \
+  -H 'X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN' \
+  --data '{
+            "title": "Test Asset",
+            "description": "How to describe the asset to our pentesters",
+            "asset_type": "web"
+          }' \
+  -v
+```
+
+For more information on each parameter, see our API reference documentation on
+how to [Create an Asset](https://docs.cobalt.io/v2/#create-an-asset).
+
+The command we use includes a `-v`, which sets up output in verbose mode. The
+command works without it. However, you would see no response from the REST call.
+
+When you review the output of the REST call with the `-v`, look for the line
+with `HTTP/2`. You'll see one of the following lines:
+<!-- The output is associated with a `201` message, which doesn't include
+results, which is why I recommend a `-v` -->
+
+| Message    | Meaning          |
+|------------|------------------|
+| HTTP/2 201 | Asset created    |
+
+For a list of error codes, see the [Errors](https://docs.cobalt.io/v2/#errors)
+section of our API reference.
+
+<!-- Maybe this table really belongs in our API reference, next to
+https://docs.cobalt.io/v2/#errors?  -->
+
+## Find Your Asset ID
+
+To add or modify information related to your asset, you'll need the asset ID.
+You can find this ID with the REST call to [Get All Assets](https://docs.cobalt.io/v2/#get-all-assets):
+
+```bash
+curl -X GET "https://api.cobalt.io/assets" \
+  -H "Accept: application/vnd.cobalt.v2+json" \
+  -H "Authorization: Bearer YOUR-PERSONAL-API-TOKEN" \
+  -H "X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN" \
+  | jq .
+```
+
+If you've set up more than one asset, you may need to search through the output.
+You can also limit the number of assets in the output with the `limit`
+parameter. For more information about each asset response field, see our API
+reference to [Get All Assets](https://docs.cobalt.io/v2/#get-all-assets).
+
+{{% alert title="Tip" color="tip" %}}
+You can use `jq` to filter assets by their `title` and `id`. To do so, end the
+REST call with `| jq -r ".detail" | .resource | .title, .id`.
+{{% /alert %}}
+
+{{% expand "Review sample output." %}}
+```json
+{
+  "data": {
+    "resource": {
+      "id": "YOUR-ASSET-IDENTIFIER",
+      "title": "Test Asset",
+      "description": "How to describe the asset to our pentesters",
+      "asset_type": "web",
+      "logo": null,
+      "attachments": []
+    },
+    "links": {
+      "ui": {
+        "url": "https://api.cobalt.io/links/<endpoint for the asset>"
+      }
+    }
+  }
+}
```

Comment:
```suggestion
```
YOUR-ASSET-TITLE
YOUR-ASSET-IDENTIFIER
```

Reply:
I chose "Test Asset" to match the name I used to create the asset.